repairs/public/js/work_order.js

```diff
@@ -1,4 +1,4 @@
-erpnext.production_order.make_se = function (frm, purpose) {
+erpnext.work_order.make_se = function (frm, purpose) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'make_se' is not in camel case. (camelcase)](https://app.codacy.com/manual/DigiThinkIT/repairs/pullRequest?prid=4737428)